src/main/java/org/jenkinsci/plugins/azurekeyvaultplugin/AzureCredentialsProvider.java

```diff
@@ -229,6 +230,27 @@ private static Collection<IdCredentials> fetchCredentials() {
                             credentials.add(cred);
                             break;
                         }
+                        case "certificate": {
+                            String passwordId = tags.get("password-id");
+                            Supplier<Secret> password = () -> Secret.fromString("");
```

Comment:
@timja 
so I made so that the password secret is initialized with an empty string 
only if the password-id tag is set, the password value would be updated by the password-id-secret-value